ccmlib/scylla_repository.py

```diff
@@ -173,6 +174,25 @@ def download_version(version, url=None, verbose=False, target_dir=None):
         tar.extractall(path=target_dir)
         tar.close()
 
+        # if relocatable package format >= 2, need to extract files under subdir
+        package_version_file = "{}/.relocatable_package_version".format(target_dir)
+        if os.path.exists(package_version_file):
+            with open(package_version_file) as f:
+                package_version = f.read().strip()
+            if package_version != '2':
+                print('Unknown relocatable package format version: ' + package_version)
+                sys.exit(1)
+            print('Relocatable package format version 2 detected.')
+            for p in Path(target_dir).iterdir():
+                if p.is_dir():
```

Comment:
there only on directory inside of it ? doesn't it has a specific name ?

Reply:
@fruch The new format contain only single directory which has same name as package (scylla-package.tar.gz -> scylla, scylla-jmx-package -> scylla-jmx ...and so on).
Why do we need this is, we want to combine all tarballs to single, unified tarball without file conflict:
https://github.com/scylladb/scylla-pkg/issues/1218
Since each package has different directory name, this script trying to find a directory in indirect way.

There is another approach, we may able to add another file to store the directory name (= package name) just like ".relocatable_package_version" file, then we don't need to do iterdir().

Reply:
if that's the reason that's fine